### PR TITLE
Adding Environment URL field in Phase model

### PR DESCRIPTION
**Pull request by vkartik97**
Added `environment_url` field in `ChallengePhase` model, that is the repository url for environment that will be used by RL Worker.
The worker can get the `environment_url` using `api/challenges/phase/environment/<phase-slug>`.

### CODE REVIEW

**Comment by vkartik97** [comment]
@RishabhJain2018 @rishabhbhardwaj @deshraj @fristonio Please review

**Comment by yashdusing** [comment]
Hey @vkartik97 ! I read your blog and was wondering if you tried [this](https://developer.nvidia.com/kubernetes-gpu) ?
Also, you mentioned of a problem regarding GPU orchestration. Could you please elaborate ?

**Comment by vkartik97** [comment]
@yashdusing 
>Hey @vkartik97 ! I read your blog and was wondering if you tried this ?

Yes, this is already enabled in [EC2 P2](https://aws.amazon.com/ec2/instance-types/p2/) but I wanted to achieve sharing of GPU resources among the environment containers in multiple pods.

> Also, you mentioned of a problem regarding GPU orchestration. Could you please elaborate ?

GPU Orchestration on P2 is working as expected, the problem is just the sharing extension. I tried it on 1 node minikube, which didn’t  give expected behaviour.

**Comment by yashdusing** [comment]
@vkartik97 I was wondering if [this](https://ray.readthedocs.io/en/latest/resources.html) can be used on top of k8s ? It has fractional gpu support and also runs on top of k8s.

**Comment by vkartik97** [comment]
@yashdusing Really thanks for researching on the topic.
I did try ray-project at the end of Phase 2, and I ran into similar this issue: [[ray] ray misuse gpu in docker container](https://github.com/ray-project/ray/issues/5245), while trying running tensorflow training on `nividia-docker`, one of the dependency for Kubernetes GPU Scheduling. I believe that the issue is due to nvidia-docker, but I need to spike more to come to any conclusion.
Again, thanks for working on this :)

File: scripts/seed.py [hunk 1]
```diff
@@ -175,11 +175,15 @@ def create_challenge_phases(challenge, number_of_phases=1):
     challenge_phases = []
```
Comment (RishabhJain2018) [inline scripts/seed.py]: Can you please open up a new PR with the changes in this file?

Comment (vkartik97) [reply scripts/seed.py]: Done

**Pull request merged**